basics/data-import/dedup.md

```diff
@@ -2,15 +2,15 @@
 description: Deduplication support in Apache Pinot.
 ---
 
-# Stream Ingestion with Dedup
+# Stream ingestion with deduplication
 
-Pinot provides native support of Deduplication during the real-time ingestion (v0.11.0+).
+Pinot provides native support for deduplication (dedup) during the real-time ingestion (v0.11.0+).
 
-To enable dedup on a Pinot table, there are a couple of table configuration and schema changes needed.
+To enable dedup on a Pinot table, make the following table configuration and schema changes.
```

Comment:
it'd be helpful to explicitly call out the necessary changes...or drop this line here and include as step 1 in procedure(s) below